src/api/app/presenters/notification_presenter.rb

```diff
@@ -73,11 +63,44 @@ def avatar_objects
     end
   end
 
+  def action_description
+    case @model.event_type
+    when 'Event::RequestStatechange', 'Event::RequestCreate', 'Event::ReviewWanted'
+      request_summary(@model.notifiable)
+    when 'Event::CommentForRequest'
+      request_summary(@model.notifiable.commentable)
+    when 'Event::CommentForProject'
+      "#{@model.notifiable.commentable.name}"
+    when 'Event::CommentForPackage'
+      commentable = @model.notifiable.commentable
+      "#{commentable.project.name} / #{commentable.name}"
+    end
+  end
+
   private
 
   def render_without_markdown(content)
     # Initializes a Markdown parser, if needed
     @remove_markdown_parser ||= Redcarpet::Markdown.new(Redcarpet::Render::StripDown)
     ActionController::Base.helpers.sanitize(@remove_markdown_parser.render(content.to_s))
   end
+
+  def kind_of_request(bs_request)
+    return 'Multiple Actions\'' if bs_request.bs_request_actions.size > 1
+
+    bs_request.bs_request_actions.first.type.titleize
+  end
+
+  def request_summary(bs_request)
+    return "#{bs_request.bs_request_actions.first.target_project}" if bs_request.bs_request_actions.size > 1
+
+    bs_request_action = bs_request.bs_request_actions.first
+
+    case bs_request_action.type
+    when 'submit', 'maintenance_incident', 'maintenance_release', 'release change_devel'
+      "#{bs_request_action.source_project} - #{bs_request_action.target_project}"
```

Comment:
hm, I think we misunderstood each other. this needs to show which package.

Reply:
True! Code updated. Please have another look, @hennevogel